src-ts/cubicons/coordinateSys.ts

```diff
@@ -1,10 +1,11 @@
 import * as d3 from "d3";
```

Comment:
I think we should get rid of `import * as d3 from "d3"` lines and import each module specifically.